### ISSUE TITLE
Missing Ubuntu eoan package

### ISSUE THREAD

**Issue by jeekajoo** [nextcloud/desktop#1518]
Sorry if it was planned to release it tomorrow when ubuntu 19.10 is released. If not, it is a reminder. Thanks a lot for all your work. I love Nextcloud. 

### Expected behaviour

Package available

### Actual behaviour

```
sudo apt install nextcloud-client
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Package nextcloud-client is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source
```

### Steps to reproduce
1. install nextcloud client on Ubuntu 19.10

### Client configuration

Operating system: Ubuntu 19.10

**Comment by jeekajoo** [comment]
Build failed : https://launchpad.net/~nextcloud-devs/+archive/ubuntu/client/+build/17824306

**Comment by jpmacedosilva** [comment]
I'm also waiting for the release of nextcloud-client for Ubuntu 19.10 Eoan.
Is there any indication as to how long it will take for this issue to be solved?

**Comment by jeekajoo** [comment]
@jpmacedosilva Meanwhile you can apt-get `nextcloud-desktop` (2.5.3-1) package that is available in Eoan.

**Comment by ivaradi** [comment]
Unfortunately a fix for the Eoan build did not make it into 2.6.0, but is in 2.6.1 (#1475). So once 2.6.1 is released, a package for Eoan should be available.

**Comment by dupondje** [comment]
I made a build with the fix in my PPA for the onces that would like it:
https://launchpad.net/~dupondje/+archive/ubuntu/ppa

Btw, whats the reason the official package is named nextcloud-desktop and the PPA package nextcloud-client? Shouldn't we align this? :)

**Comment by ivaradi** [comment]
@dupondje thanks for the build. 

I have just created a package of 2.6.0 for Eoan with the fix into the "official" repo, so it is now possible to go back to that one. Hopefully, from now on there will be no problem with Eoan or any other Ubuntu versions.

Regarding the naming of the package: originally it was nextcloud-client, and I just kept the name. But you are right, it would be more consistent to name it nextcloud-desktop, so I am going to rename the package so, and provide a transitional package to allow for smoothless upgrade.
